src/cplus_plugin/models/base.py

```diff
@@ -402,6 +402,7 @@ def from_dict(cls, activity_dict: typing.Dict):
             del pathway["carbon_uuids"]
             pathways.append(NcsPathway(**pathway))
         activity_dict["pathways"] = pathways
+        del activity_dict["mask_uuids"]
```

Comment:
Thanks for changes @zamuzakki, one question, where are the `mask_uuids` from?

Reply:
@Samweli it's coming from the API.

Reply:
@zamuzakki what if the key 'mask_uuids' is missing?

Reply:
@Samweli It could result in KeyError. I've updated it to `activity_dict.pop("mask_uuids", None)`